lux/extensions/rest/models.py

```diff
@@ -138,9 +140,11 @@ def field_options(self, request):
             target = self.get_target(request)
             yield 'data-remote-options', json.dumps(target)
             yield 'data-remote-options-id', self.id_field
-            yield 'data-remote-options-value', self.repr_field
+            yield 'data-remote-options-value', json.dumps({
+                'type': 'field',
+                'source': self.repr_field})
             yield 'data-ng-options', self.remote_options_str.format(
-                id=self.id_field, repr=self.repr_field, options=self.api_name)
```

Comment:
No particular reason, I just went with leaving the replacement fields in remote_options_str. I've changed it now.